### PR TITLE
gh-124 Fix E2E tests

### PR DESCRIPTION
**Pull request by ghillert**
Resolves https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/124

FYI: I was having a hard time getting button clicks to work. Our loading spinner seems to interfere in strange ways. One button click I even had to comment out. 

Ideally, I was hoping to eliminate "sleeps" altogether, and rather work with "once button abc becomes available, do xyz ...".

Strangely enough, I could not find a workable solution. Maybe something to revisit once we think about moving towards Angular2.

### CODE REVIEW

**Comment by BoykoAlex** [comment]
I wasn't able to start e2e tests. I have updated the se,enium chrome driver but the tests failed to start due to:
```
Error: Could not find chromedriver at /Users/aboyko/WebstormProjects/spring-cloud-dataflow-ui/ui/node_modules/protractor/selenium/chromedriver
```
The file appears to be named `chromedriver_2.21` for me instead of `chromedriver`. However, once i manually renamed the file the tests ran and all passed.
I've tried updating the version for `grunt-protractor-runner` to `3.2.0' from `2.0.0` but that didn't help.
@ghillret Do the tests run for you ok after `mvn clean install`? What is the chrome driver file name for you?

File: ui/package.json [hunk 1]
```diff
@@ -33,18 +33,18 @@
...
-    "karma-jasmine": "0.3.5",
+    "karma-jasmine": "0.3.8",
     "karma-coffee-preprocessor": "0.2.1",
     "karma-requirejs": "0.2.2",
     "karma-phantomjs-launcher": "0.1.4",
     "karma": "0.12.31",
     "karma-ng-html2js-preprocessor": "0.1.2",
     "phantomjs": "1.9.16",
-    "protractor": "2.0.0",
+    "protractor": "3.2.2",
```
Comment (BoykoAlex) [inline ui/package.json]: Maybe we should keep the protractor at 2.x.x for now... The Chrome driver new protractor downloads after unzipping etc. has the name `chromedriver_2.21` while protractor expects it to be `chromedriver` hence e2e tests cannot be run.
If `protractor` version becomes `3.x.x` it's possible we should bump up `grunt-protractor-runner` to `3.x.x`  too.

**Comment by ghillert** [comment]
@BoykoAlex Please retry I reverted the protractor version.

**Comment by BoykoAlex** [comment]
Merged with commit #594b687

**Pull request closed without merging**